test/e2e/tests/test_external_models.py

```
@@ -0,0 +1,371 @@
+RECONCILE_WAIT = int(os.environ.get("E2E_RECONCILE_WAIT", "12"))
+TLS_VERIFY = os.environ.get("E2E_SKIP_TLS_VERIFY", "").lower() != "true"
+
+EXTERNAL_MODEL_NAME = "e2e-external-model"
+
+
+# ─── Helpers ─────────────────────────────────────────────────────────────────
+
+def _apply_cr(cr_dict: dict):
+    """Apply a Kubernetes CR from a dict."""
+    result = subprocess.run(
+        ["oc", "apply", "-f", "-"],
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)